### DISCUSSION TITLE
welchen Sensor benutzen auf raspberry pi5 und Bookworm

### DISCUSSION THREAD

**Discussion by marco050969** [AllskyTeam/allsky-modules#51] [Q&A] [answered]
Hi, 
welchen Sensor sollte ich kaufen für das Dew Heater Modul von Allsky?
Ich hatte bisher auf pi4 und Bullseye einen HTU21 benutzt, dieser funktioniert aber nicht mehr unter Bookworm und PI5.
Die Gpio Pinbelegung hat sich auch geändert und die Software läuft nicht mehr auf Bookworm, neue gibt es nicht mehr, da schon 7 Jahre alt.
Hat irgendwer eine Empfehlung und eine Anleitung, wie ich das ganze einbinden und programmieren muss, das ich mit dem Sensor eine Heizung steuern kann über das Modul von Allsky.

**Comment by Alex-developer** [comment, marked as answer]
There is a module available for Allsky to manage a dew heater. Currently this module supports the following sensors

SHT31 - i2c device
DHT22 - 1 wire device 
DHT11 - 1 wire device
AM2302 - 1 wire device
BME280-I2C - i2c device
HTU21 - i2c device
AHTx0 - i2c device (i2c version of the dht22)
SOLO-Cloudwatcher - DO NTO USE

Personally I use the SHT31 and HTU21 as I have found them to be the most accurate but any of the above will work fine

**Reply by marco050969** [reply]
Hi Alex,

the htu21/ and relay works here on a Pi5 with Bookworm perfect.

Thanks

**Reply by Alex-developer** [reply]
Glad you have it working :-)

**Comment by Alex-developer** [comment]
> Hi Alex,
> 
> the htu21/ and relay works here on a Pi5 with Bookworm perfect.
> 
> Thanks
